Host a Static Website with Hugo and NGINX.md

```diff
@@ -141,7 +140,7 @@ Enter the `username` that you use to log into your Linode server. This user must
 Enter the `IPAddress` of your Linode server.
 {{< /note>}}
 
-Further clarification on logging in with SSH can be found in the [Getting started](/docs/getting-started) guide in the section titled *Log In for the First Time*. Create a directory named hugo and change into this directory. 
+Further clarification on logging in with SSH can be found in the [Getting started](/docs/getting-started) guide in the section titled *Log In for the First Time*. Create a directory named "hugo" and change into this directory. 
```

Comment:
Also perhaps "Log In for the First Time" should be in double quotes for consistency.

Reply:
This statement migrated into the wrong place during the updates. It needed to be in the "Installing Hugo" section.